packages/web/src/features/notice/componenets/NoticePagination.tsx

```diff
@@ -0,0 +1,131 @@
+"use client";
+
+import React from "react";
+import styled from "styled-components";
+
+import Icon from "@sparcs-clubs/web/common/components/Icon";
+
+interface NoticePaginationProps {
+  totalPage: number;
+  currentPage: number;
+  limit: number;
+  setPage: React.Dispatch<number>;
+}
+
+const NoticePaginationInner = styled.div`
+  flex-basis: auto;
+  flex-grow: 0;
+  flex-shrink: 0;
+  display: flex;
+  flex-direction: row;
+  column-gap: 16px;
+`;
+
+// 버튼 스타일 지우기 위해 구성했는데, 올바른 방법인지 잘 모르겠습니다.
+const ButtonWrapper = styled.button`
+  border: none;
+  padding: 0;
+  background-color: ${({ theme }) => theme.colors.WHITE};
+`;
+
+const WalkableIndex = styled.div`
+  font-weight: ${({ theme }) => theme.fonts.WEIGHT.MEDIUM};
+  font-size: 16px;
+  line-height: 20px;
+`;
+
+const CurrentIndex = styled.div`
+  font-weight: ${({ theme }) => theme.fonts.WEIGHT.MEDIUM};
+  font-size: 16px;
+  line-height: 20px;
+  color: ${({ theme }) => theme.colors.PRIMARY};
+`;
```

Comment:
제가 pagination 디자인을 조금 고쳐서.. 요것두 그냥 같이 반영해서 넣어주세용

```suggestion
const WalkableIndex = styled.div`
  width: 20px;
  font-weight: ${({ theme }) => theme.fonts.WEIGHT.MEDIUM};
  font-size: 16px;
  line-height: 20px;
  color: ${({ theme }) => theme.colors.BLACK};
  text-align: center;
`;

const CurrentIndex = styled.div`
  width: 20px;
  font-weight: ${({ theme }) => theme.fonts.WEIGHT.MEDIUM};
  font-size: 16px;
  line-height: 20px;
  color: ${({ theme }) => theme.colors.PRIMARY};
  text-align: center;
`;
```